frontend/src/stimulus/controllers/dynamic/meeting-content.controller.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * -- copyright
+ * OpenProject is an open source project management software.
+ * Copyright (C) 2023 the OpenProject GmbH
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License version 3.
+ *
+ * OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+ * Copyright (C) 2006-2013 Jean-Philippe Lang
+ * Copyright (C) 2010-2013 the ChiliProject Team
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * See COPYRIGHT and LICENSE files for more details.
+ * ++
+ */
+
+import { Controller } from '@hotwired/stimulus';
+
+export default class MeetingContentController extends Controller {
+  static targets = [
+    'renderedText',
+    'editor',
+    'attachments',
+    'editButton',
+  ];
+
+  initialize() {
+    // This should have been achieved by the `data-loader-edit-state-value` attribute, but for unknown reasons
+    // this does not apply the value to the variable. Doing it manually for now.
+    const rawInitialValue = this.element.getAttribute('data-loader-edit-state-value');
+
+    if (rawInitialValue === 'true') {
+      this.editStateValue = true;
+    } else if (rawInitialValue === 'false') {
+      this.editStateValue = false;
+    }
```

Comment:
I added this workaround for what should have worked out of the box. I actually applied the `data-loader...` attribute successfully for the `admin-roles.controller` and cannot spot any difference. Do you have an idea, @oliverguenther

Reply:
The values are not called `data-loader`, but `data-[controllername]-[valuename]`. so this would be `data-meeting-content-state-value="..."`.  That's why it wasn't picked up

cf., https://stimulus.hotwired.dev/reference/values#properties-and-attributes

Reply:
Ah, I was fooled by the example on that page (it is called loader which sounded like what I was trying to achieve, load data). 
Thanks @oliverguenther